extensions/wikidata/src/org/openrefine/wikidata/qa/scrutinizers/SingleValueScrutinizer.java

```diff
@@ -45,17 +47,28 @@
 public class SingleValueScrutinizer extends EditScrutinizer {
 
     public static final String type = "single-valued-property-added-more-than-once";
-    public static String SINGLE_VALUE_CONSTRAINT_QID = "Q19474404";
-    public static String SINGLE_BEST_VALUE_CONSTRAINT_QID = "Q52060874";
+    public String singleValueConstraintQid;
+    public String singleBestValueConstraintQid;
+
+    @Override
+    public boolean prepareDependencies() {
+        SingleValueConstraint singleValueConstraint = constraints.getSingleValueConstraint();
+        SingleBestValueConstraint singleBestValueConstraint = constraints.getSingleBestValueConstraint();
+        if (singleValueConstraint == null) return false;
+        if (singleBestValueConstraint == null) return false;
+        singleValueConstraintQid = singleValueConstraint.getQid();
+        singleBestValueConstraintQid = singleBestValueConstraint.getQid();
+        return false;
```

Comment:
Doesn't this mean that this will never get used?

Reply:
This is obviously a mistake :)